engine/installation/linux/docker-ce/debian.md

```diff
@@ -26,6 +26,7 @@ and distributions for different Docker editions, see
 To install Docker CE, you need the 64-bit version of one of these Debian or
 Raspbian versions:
 
+- Buster (edge only)
```

Comment:
Wondering if we should add versions to all of these;

- Buster 10 (or `Debian 10 "Buster"`? Not sure what the preferred notation is)
- Stretch 9
- Jessie 8 (LTS)
- Wheezy 7.7 (LTS)

Frankly; we should start thinking of removing Wheezy (as it's getting old in the tooth), and it won't even support docker without installing a custom kernel

Reply:
I agree .. but there has been debate about that. I'll just do it. More clarity is always better than less.

Reply:
Ignore the "deprecate wheezy" part, that's just me rambling 😅 - plans were already in place for that, but will be early next year probably